src/actions/query.js

```diff
@@ -73,6 +73,21 @@ export function updateQuery(id, data) {
     };
 }
 
+export function removeQuery(id) {
+    return ({ dispatch, getState, z }) => {
+        const orgId = getState().org.activeId;
+        const where = `orgs/${orgId}/people/queries/${id}`.split('/');
```

Comment:
No need to `split()` and `...spread`. You can just pass the string directly to `z.resource()`. See README for the Zetkin SDK here: https://github.com/zetkin/zetkin.js